### DISCUSSION TITLE
Request to include HPO obo file in the

### DISCUSSION THREAD

**Discussion by mano-at-sdsc** [nih-cfde/published-documentation#342] [Feature Requests] [answered]
I  got it from https://hpo.jax.org/app/download/ontology (http://purl.obolibrary.org/obo/hp.obo), but may be it can be included in the file external_CV_reference_files.zip in the same was as doid obo flat file is included. Thanks.

**Comment by abradyIGS** [comment, marked as answer]
The one we're using is posted (along with the newest version of the submission prep script) [here](https://osf.io/bq6k9/) -- please let me know if you need it updated.

That whole folder has been updated as of today: please re-download the submission prep script and ontology files (the latter are found in the "external_cv_reference_files" subfolder).

The master C2M2 JSON schema has also been updated and an up-to-date copy will need to be included with your submissions; please make sure you have the newest version ([permalink](https://osf.io/c63aw/)).

**Comment by mano-at-sdsc** [comment]
Thanks, Arthur.

Mano

**Comment by mano-at-sdsc** [comment]
Please check the URL for JSON schema at this page:

https://github.com/nih-cfde/published-documentation/wiki/Quickstart

In the sentence (in the cfde-submit subsection):

**To use the tool, give it the path to the directory containing all 22 tables plus the [required JSON schema file](https://osf.io/e5tc2/):**

https://osf.io/e5tc2/ is obsolete now.

The JSON link here is updated: https://docs.nih-cfde.org/en/latest/c2m2/draft-C2M2_specification/

Mano

**Comment by abradyIGS** [comment]
Updated the offending page. Thanks Mano!
